documentation/wiki/ChangeWaves-Dev.md

```diff
@@ -1,12 +1,10 @@
 ⚠ This doc is intended for internal teams.
 
 # What are Change Waves?
-A Change Wave is a set of risky features developed under the same opt-out flag. This flag happens to be the version of MSBuild that the features were developed for. The purpose of this is to warn developers of risky changes that will become standard functionality down the line. 
```

Comment:
I actually thought the previous version was perfect here. Other than ending a sentence with a preposition 😛

Reply:
At this point I'm likely overthinking it. Will revert to the original.